Documentation/Installation/Install.rst

```diff
@@ -13,10 +13,11 @@ Welcome to the TYPO3 installation guide. This guide covers each of the steps req
 Installation And Deployment
 ---------------------------
 
-Composer should only ever be used to install TYPO3 on a local or development environment. By not using Composer in a live environment, the need to install
-dependencies are completely removed - reducing potential vectors for attack.
+Composer should only ever be used to install TYPO3 on a local or development environment.
+By **not using Composer** in a live environment there will be no need for installing
+dependencies - reducing potential vectors for attack.
```

Comment:
The whole part is still very confusing. Not sure how Composer on the server should increase attack vectors at all. But it's a good practise to use Composer locally or on the deployment environment only, that's correct so far.

Reply:
Probably not composer "per se" - but composer on the server would tempt people to install dev dependencies or do requires and testing directly on the live system.

Reply:
Composer fetches software from the internet. When a software update is made, then the dependent extensions will be fetched. The server with the software sources could be compromised. 
Or the composer file could be manipulated without any notice. 

If you copy everything from your local computer, then this should be safer. Then you can be sure, that both have the same software. Your local computer is not exposed to the internet attackers.

Reply:
> Probably not composer "per se" - but composer on the server would tempt people to install dev dependencies or do requires and testing directly on the live system.

Agree, but that's not was is written here.

Reply:
I tried to make the suggestions more specific here. please see if we can get it merged like this now.